app/src/main/java/com/manna/presentation/sign_up/ServiceTermsFragment.kt

```diff
@@ -50,10 +43,42 @@ class ServiceTermsFragment :
                     ContextCompat.getDrawable(requireContext(), R.drawable.bg_btn_gray)
             }
         }
+
+        binding.tvCheck.setOnClickListener {
+            if (binding.cbCheck.isChecked) {
+                binding.cbCheck.isChecked = false
+                binding.tvSignUp.isEnabled = false
+                binding.tvSignUp.background =
+                    ContextCompat.getDrawable(requireContext(), R.drawable.bg_btn_gray)
+            } else {
+                binding.cbCheck.isChecked = true
+                binding.tvSignUp.isEnabled = true
+                binding.tvSignUp.background =
+                    ContextCompat.getDrawable(requireContext(), R.drawable.bg_btn_blue)
+            }
+        }
+
         binding.tvSignUp.setOnClickListener {
             startActivity(HomeActivity.getIntent(requireContext(), isWelcome = true))
             requireActivity().finish()
         }
+
+        binding.run {
+            layoutTitleBar.tvTitle.text = getString(R.string.sign_up_service_terms_title)
+            layoutTitleBar.ivBack.setOnClickListener {
+                onBackPressed()
+            }
+            tvContent.movementMethod = ScrollingMovementMethod()
+        }
+    }
+
+    private fun onBackPressed() {
+        val fragment =
+            parentFragmentManager.fragments.findLast { it is BaseFragment<*> && it !== this }
+        if (fragment != null) {
+            parentFragmentManager.beginTransaction().show(fragment).commit()
+        }
+        parentFragmentManager.beginTransaction().remove(this@ServiceTermsFragment).commit()
```

Comment:
```suggestion
                parentFragmentManager.beginTransaction()
                    .apply {
                        if (fragment != null) {
                            show(fragment)
                        }
                    }
                    .remove(this@ServiceTermsFragment)
                    .commit()

```
이런식으로 커밋한번으로도 가능은 해요!